test/lib/MangroveTest.sol

```diff
@@ -497,17 +497,17 @@ contract MangroveTest is Test2, HasMgvEvents {
   }
 
   function logTickTreeBranch(IMangrove _mgv, OLKey memory _ol) internal view {
-    console.log("--------CURRENT bin TREE BRANCH--------");
+    console.log("--------CURRENT tick tree BRANCH--------");
     MgvStructs.LocalPacked _local = _mgv.local(_ol);
     Bin bin = _local.bestBin();
     console.log("Current bin %s", toString(bin));
     console.log("Current posInLeaf %s", bin.posInLeaf());
     int leafIndex = bin.leafIndex();
     console.log("Current leaf %s (index %s)", toString(_mgv.leafs(_ol, leafIndex)), vm.toString(leafIndex));
-    console.log("Current level 0 %s (index %s)", toString(_local.level2()), vm.toString(bin.level2Index()));
-    int level1Index = bin.level1Index();
-    console.log("Current level 1 %s (index %s)", toString(_mgv.level1(_ol, level1Index)), vm.toString(level1Index));
-    console.log("Current level 2 %s", toString(_local.level0()));
+    console.log("Current level 3 %s (index %s)", toString(_local.level3()), vm.toString(bin.level3Index()));
+    int level2Index = bin.level2Index();
+    console.log("Current level 2 %s (index %s)", toString(_mgv.level2(_ol, level2Index)), vm.toString(level2Index));
+    console.log("Current level 1 %s", toString(_local.level1()));
```

Comment:
Is root missing here?

Reply:
fixed in b629e86b9d32fcd905bf8643ef65408841751c08